truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity 0.5.10;
+
+import './Stoppable.sol';
+
+contract Splitter is Stoppable {
+    
+    struct Account {
+        uint    pendingWithdrawals;
+    }
+    
+    mapping(address => Account) accountStorage;
+
+    event LogEthSent(address sender, uint value, address receiver1, address receiver2);
+    event LogWithdrawed(address sender, uint amount);
+    
+    constructor() public payable {}
+    
+    function splitEth(address receiver1, address receiver2) public _onlyIfRunning payable returns(bool) {
+        require(msg.value <= msg.sender.balance, 'The sender balance shoud be higher the ether\'s value');
+
+        accountStorage[receiver1].pendingWithdrawals += msg.value / 2;
+        accountStorage[receiver2].pendingWithdrawals += msg.value / 2;
+
+        emit LogEthSent(msg.sender, msg.value, receiver1, receiver2);
+        
+        return true;
+    }
+    
+    function getBalance() public view returns(uint256) {
```

Comment:
This sort of thing can be useful when playing around in Remix because they don't have a good account explorer to see what's going on. In production, it's unnecessary because everyone can everyone's balance at all times. Not sure if this is a left over from dev/test or deliberately here. In any case, every opportunity to simplify and shrink is a good thing to do.

Reply:
Indeed, I added this functionality cause remix does not allow me to get the balance otherwise. I will delete it >)